save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/validation/FrontendRoutes.kt

```diff
@@ -39,6 +39,7 @@ enum class FrontendRoutes(val path: String) {
     SETTINGS_ORGANIZATIONS("$SETTINGS/organizations"),
     SETTINGS_PROFILE("$SETTINGS/profile"),
     SETTINGS_TOKEN("$SETTINGS/token"),
+    SIGNLE_VULNERABILITY("$VULN/collection"),
```

Comment:
Should `SINGLE_VULNERABILITY` be `/collection`?

Reply:
no, why? 
/vuln/collection, because it's in /vuln domain

Reply:
I mean, _collection_ is not about _single_ vulnerability

Reply:
> I mean, _collection_ is not about _single_ vulnerability

ah, got it. Anyway, users will not see it on UI (only search panel): they will see vuln/list/NAME